### ISSUE TITLE
Mana Bar, Spells, and MP stats disappeared after class change

### ISSUE THREAD

**Issue by nataliezook** [HabitRPG/habitica#3600]
A while back I made a class change from Rogue (lvl 50-something) to Warrior, and then quickly again to Mage. (I only wanted to buy up Warrior gear.) I noticed my mana and mana stats vanished when I first changed; the mana bar is simply not there as if I were under lvl 10, and the stats page is rearranged without the option to delegate MP also as if I were under lvl 10. They've not shown back up, and I don't see my class spells available for purchase.

This was about two weeks ago and coincided with a family emergency that required me to take a break from Habit, hence why I am only reporting it now.

Username: UnconsciousPlots
ID #: ec6e08f5-f39d-47f8-95e1-7909cd18965e
Class: Mage
Level: 57

**Comment by deilann** [comment]
Sounds like you accidentally opted out of classes instead. Go to Setting >
Site and click the button that says to enable classes

**Comment by nataliezook** [comment]
AHA. Now I understand what happened. Interesting quirk here involving multiple tabs. Let me see if I can describe it properly, since I don't think what happened is supposed to happen.

1) I won 3 gems in a party challenge, which I decide to use for a class change. I go ahead and click the option for the class changing, pulling up the pop-up box with class choices.
2) I realize I want to ask my party's opinion on which class would be the most useful to the party, so I open up a second HabitRPG tab intending to go to my party's chat, but the pop-up box is still there in the second tab. I hit "opt-out," defaulting to warrior, in order to go to my party's chat. The original tab is still open with its pop-up box. I take the opportunity to buy up warrior gear while I await party member responses.
3) After my party has given their input, I've made my decision to go Mage. Not knowing exactly where the "opt in" is, I decide it is easiest to return to my original tab that still has its pop-up class choice window. I select Mage, and am able to buy Mage gear, but as I described, the mana options don't appear because I have unwittingly simultaneously opted out of the class system and been able to choose a class other than default. Confusion ensues. XD

Thanks. I hadn't even thought to check if I was opted out.

EDIT: After a little experimentation with the "opt-in" button and opening new tabs, I've found this bug lets me fully change classes indefinitely for free once I'm in an opted out state.
